Prace_domowe/Praca_domowa3/Grupa4/Marcin Wilk/PD3.ipynb

```diff
@@ -0,0 +1,1085 @@
+{
```

Comment:
<p>Dosyć nietypowy podział. Default to 80(train) i 20 (test). Jakieś uzasadnienie?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I chyba jest na odwrót</p>

Reply:
<p>Wziąłem 60(train) i 40(test), w jednym wykładzie jak patrzyłem był dopuszczany taki podział, a na innym przedmiocie braliśmy nawet 50-50, więc stwierdziłem, że 60-40 jak najbardziej ma sens</p>